modules/light-clients/07-tendermint/update_test.go

```diff
@@ -557,6 +557,61 @@ func (suite *TendermintTestSuite) TestUpdateState() {
 	}
 }
 
+func (suite *TendermintTestSuite) TestUpdateStateCheckTx() {
```

Comment:
also checked this fails without the if statement added to the pruning logic

Reply:
nice test! 👍🏻